.travis.yml

```diff
@@ -21,9 +21,6 @@ before_script:
 script:
   - go build -v ./cmd/dep
   - go vet $PKGS
-  # Ignore the deprecation warning about filepath.HasPrefix (SA1019). This flag
-  # can be removed when issue #296 is resolved.
-  - staticcheck -ignore='github.com/golang/dep/context.go:SA1019 github.com/golang/dep/cmd/dep/init.go:SA1019' $PKGS
```

Comment:
This removes staticcheck as a whole - we need just staticcheck without the flags. so, `staticcheck $PKGS`